package.json

```
@@ -0,0 +1,106 @@
+{
+	"name": "richdocuments",
+	"description": "Collabora online integration",
+	"version": "1.0.0",
+	"authors": [
+		{
+			"name": "Julius Härtl",
+			"email": "jus@bitgrid.net",
+			"role": "Developer"
+		}
+	],
+	"license": "agpl",
```

```suggestion
	"license": "AGPL-3.0-or-later",
```
You can put the full license in `package.json`.